### PR TITLE
Add links to ranking pages from place overview table

### PR DESCRIPTION
**Pull request by beets**
* Removed support for multiple stat var (since it's not used at the moment)
* Fully specify params in the url path (changed the URL pattern as well)
* Cleaned up some unused functions
* Added error handling
* Move some static page rendering to the client

### CODE REVIEW

File: static/js/ranking/ranking_page.tsx [hunk 1]
```diff
@@ -20,70 +20,69 @@ import { STATS_VAR_TITLES } from "../shared/stats_var_titles";
...
+      </div>
+    );
   }
 
   componentDidMount() {
-    const statsParam = this.props.statVars.map((x) => `stat=${x}`).join("&");
     return axios
       .get(
-        `/ranking/api/${this.props.placeType}/${this.props.withinPlace}?${statsParam}`
+        `/ranking/api/${this.props.statVar}/${this.props.placeType}/${this.props.withinPlace}`
```
Comment (shifucun) [inline static/js/ranking/ranking_page.tsx:72]: should this be /api/ranking/...?

**Comment by beets** [comment]
It's currently in the same ranking.py handler - I didn't want to add
another module just for one API call, but I can move it if needed for
consistency.

**Comment by beets** [comment]
moved the handler, ptal!

**Review by shifucun — APPROVED** [review]
thanks for the change!

**Pull request merged**